app/lib/bing_api_v7.rb

```diff
@@ -0,0 +1,47 @@
+require 'net/https'
+require 'uri'
+require 'json'
+
+class BingApiV7
+  BING_URI = 'https://api.cognitive.microsoft.com'.freeze
+  BING_PATH = '/bing/v7.0/search'.freeze
+
+  def initialize(company)
+    @company = company
+  end
+
+  def uri
+    URI(BING_URI + BING_PATH + "?q=" + CGI.escape(@company.domain))
```

Comment:
move this "@company.domain" and parameter to initialize and use here.
we will need to make other searches different than just company domain.